clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -445,14 +467,7 @@ class AutoScalingWorker {
     for (appAccountRegion in applicationAccountRegions) {
       if (appAccountRegion && appAccountRegion.contains(":")) {
         def (app, account, regions) = appAccountRegion.split(":")
```

Comment:
I still think we need to wrap all this in a try/catch ... in this case we're looking for a `:` but really needing there to be 2 (otherwise `def (app, account, regions) = appAccountRegion.split(":")` is going to fail).

Reply:
Yes, sorry I forgot to add this and will update to include this